starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -622,9 +622,6 @@ sub check_dither {
     my $obs_end_pad = 3*60;
     my $manvr;
 
-    if ( $self->{obsid} =~ /^\d*$/){
-	return if ($self->{obsid} >= $ER_MIN_OBSID); # For eng obs, don't have OR to specify dither
```

Comment:
This is basically just moved later in the method so that the cmd_dither_* values are set even for ERs.